src/main/kotlin/domain/card/Cards.kt

```diff
@@ -0,0 +1,67 @@
+package domain.card
+
+class Cards(cards: List<Card> = emptyList()) {
+
+    private val cards: MutableList<Card> = cards.toMutableList()
+
+    fun add(card: Card) {
+        cards.add(card)
+    }
+
+    fun current(): Set<Card> {
+        return cards.toSet()
+    }
+
+    fun result(): Int {
+        val sumWithoutAce = sumWithoutAce()
+
+        if (doesNotHaveAce()) {
+            return sumWithoutAce
+        }
+
+        val sumOfAllCases = sumOfAceCases().map {
+            sumWithoutAce + it
+        }
+
+        if (sumOfAllCases.contains(BLACKJACK_POINT)) {
```

Comment:
contains가 infix 함수이니 만큼 다음과 같이 표현할 수 있을 것 같아요.
```suggestion
        if (BLACKJACK_POINT in sumOfAllCases) {
```